src/explore-education-statistics-admin/src/pages/release/components/ReleaseSummaryForm.tsx

```diff
@@ -102,7 +102,13 @@ const ReleaseSummaryForm = <
 
   const baseSchema = Yup.object<ReleaseSummaryFormValues>({
     timePeriodCoverageCode: Yup.string().required('Choose a time period'),
-    timePeriodCoverageStartYear: Yup.string().required('Enter a year'),
+    timePeriodCoverageStartYear: Yup.string()
```

Comment:
There's currently no unit tests for this form component, so let's write some.

Reply:
added tests:
  ReleaseSummaryForm
    √ renders correctly (104ms)
    √ validation error render where appropriate (278ms)
    √ renders with provided initial values (31ms)
    √ submits form with valid values (133ms)